pb/models.py

```diff
@@ -158,19 +158,23 @@ class Meta:
 
     @staticmethod
     def get_event(obj):
-        return obj.IRBEVENT[0].EVENT
+        if obj is not None and hasattr(obj, 'IRBEVENT'):
+            return obj.IRBEVENT[0].EVENT
 
     @staticmethod
     def get_event_id(obj):
-        return obj.IRBEVENT[0].EVENT_ID
+        if obj is not None and hasattr(obj, 'IRBEVENT'):
+            return obj.IRBEVENT[0].EVENT_ID
 
     @staticmethod
     def get_status(obj):
-        return obj.IRB_STATUS[0].STATUS
+        if obj is not None and hasattr(obj, 'IRB_STATUS'):
+            return obj.IRB_STATUS[0].STATUS
 
     @staticmethod
     def get_status_id(obj):
-        return obj.IRB_STATUS[0].STATUS_ID
+        if obj is not None and hasattr(obj, 'IRB_STATUS'):
+            return obj.IRB_STATUS[0].STATUS_ID
```

Comment:
The error we had was:
    return obj.IRB_STATUS[0].STATUS_ID
AttributeError: 'NoneType' object has no attribute 'IRB_STATUS'

Which means that there was and obj, and that obj has an IRB_STATUS, and that array contained an element at [0], BUT the element at [0] was null, so I'm not sure this will actually correct for the problem, but maybe it does.

Reply:
Hmm...
I read that the other way.
NoneType refers to obj

Reply:
@danfunk ^^^^^^ What are your thoughts about my comment? Is this OK to merge?